--- Makefile
+++ Makefile
@@ -50,13 +50,13 @@
 .PHONY: lib lib-release liblz4.a
 lib: liblz4.a
 lib lib-release liblz4.a:
-	@$(MAKE) -C $(LZ4DIR) $@
+	@$(MAKE) -C $(LZ4DIR) CFLAGS="$(CFLAGS)" $@
 
 .PHONY: lz4 lz4-release
 lz4 : liblz4.a
 lz4-release : lib-release
 lz4 lz4-release :
-	@$(MAKE) -C $(PRGDIR) $@
+	@$(MAKE) -C $(PRGDIR) CFLAGS="$(CFLAGS)" $@
 	@cp $(PRGDIR)/lz4$(EXT) .
 
 .PHONY: examples
