--- src/locate.c
+++ src/locate.c
@@ -973,7 +973,13 @@
       if (stat (entry, &st) != 0)
 	{
 	  if (conf_quiet == false)
-	    error (0, errno, _("can not stat () `%s'"), entry);
+            {
+              if (errno == ENOENT)
+                error (0, 0, _("database not found. Please run `%smupdatedb' to create it."),
+                       getuid() ? "sudo " : "");
+              else
+                error (0, errno, _("can not stat () `%s'"), entry);
+            }
 	  goto err;
 	}
       if (!db_is_privileged (&st))
--- tests/locate.at
+++ tests/locate.at
@@ -613,8 +613,8 @@
 AT_KEYWORDS([locate])
 
 AT_CHECK([locate -d db '*' 2>err], 1)
-AT_CHECK([sed "s/': .*/': /" < err], ,
-[locate: can not stat () `db': @&t@
+AT_CHECK([sed "s/sudo //" < err], ,
+[locate: database not found. Please run `mupdatedb' to create it.
 ])
 
 > db
