--- ChangeLog
+++ ChangeLog
@@ -1133,8 +1133,8 @@
 
 2004-08-19  Martin Mares  <mj@ucw.cz>
 
-	* pciutils.spec (%attr): Setting attributes for /sbin/* is an anachronism,
-	because pciutils by default install to /usr/sbin. Update.
+	* pciutils.spec (%attr): Setting attributes for /bin/* is an anachronism,
+	because pciutils by default install to /usr/bin. Update.
 
 2004-08-13  Martin Mares  <mj@ucw.cz>
 
--- Makefile
+++ Makefile
@@ -33,7 +33,7 @@
 
 # Installation directories
 PREFIX=/usr/local
-SBINDIR=$(PREFIX)/sbin
+BINDIR=$(PREFIX)/bin
 SHAREDIR=$(PREFIX)/share
 IDSDIR=$(SHAREDIR)
 MANDIR:=$(shell if [ -d $(PREFIX)/share/man ] ; then echo $(PREFIX)/share/man ; else echo $(PREFIX)/man ; fi)
@@ -118,9 +118,9 @@
 
 install: all
 # -c is ignored on Linux, but required on FreeBSD
-	$(DIRINSTALL) -m 755 $(DESTDIR)$(SBINDIR) $(DESTDIR)$(IDSDIR) $(DESTDIR)$(MANDIR)/man8 $(DESTDIR)$(MANDIR)/man7 $(DESTDIR)/$(MANDIR)/man5
-	$(INSTALL) -c -m 755 $(STRIP) lspci setpci $(DESTDIR)$(SBINDIR)
-	$(INSTALL) -c -m 755 update-pciids $(DESTDIR)$(SBINDIR)
+	$(DIRINSTALL) -m 755 $(DESTDIR)$(BINDIR) $(DESTDIR)$(IDSDIR) $(DESTDIR)$(MANDIR)/man8 $(DESTDIR)$(MANDIR)/man7 $(DESTDIR)/$(MANDIR)/man5
+	$(INSTALL) -c -m 755 $(STRIP) lspci setpci $(DESTDIR)$(BINDIR)
+	$(INSTALL) -c -m 755 update-pciids $(DESTDIR)$(BINDIR)
 	$(INSTALL) -c -m 644 $(PCI_IDS) $(DESTDIR)$(IDSDIR)
 	$(INSTALL) -c -m 644 lspci.8 setpci.8 update-pciids.8 $(DESTDIR)$(MANDIR)/man8
 	$(INSTALL) -c -m 644 pcilib.7 $(DESTDIR)$(MANDIR)/man7
@@ -154,7 +154,7 @@
 endif
 
 uninstall: all
-	rm -f $(DESTDIR)$(SBINDIR)/lspci $(DESTDIR)$(SBINDIR)/setpci $(DESTDIR)$(SBINDIR)/update-pciids
+	rm -f $(DESTDIR)$(BINDIR)/lspci $(DESTDIR)$(BINDIR)/setpci $(DESTDIR)$(BINDIR)/update-pciids
 	rm -f $(DESTDIR)$(IDSDIR)/$(PCI_IDS)
 	rm -f $(DESTDIR)$(MANDIR)/man8/lspci.8 $(DESTDIR)$(MANDIR)/man8/setpci.8 $(DESTDIR)$(MANDIR)/man8/update-pciids.8
 	rm -f $(DESTDIR)$(MANDIR)/man7/pcilib.7
--- lib/aix-device.c
+++ lib/aix-device.c
@@ -22,7 +22,7 @@
 
 #include "internal.h"
 
-#define AIX_LSDEV_CMD "/usr/sbin/lsdev -C -c bus -t pci\\* -S available -F name"
+#define AIX_LSDEV_CMD "/usr/bin/lsdev -C -c bus -t pci\\* -S available -F name"
 #define AIX_ODMGET_CMD \
   "/usr/bin/odmget -q 'name=%s and attribute=bus_number' CuAt | \
    /usr/bin/awk '$1 == \"value\" { print $3 }'"
--- lib/configure
+++ lib/configure
@@ -19,11 +19,11 @@
 	sys=`uname -s`
 	rel=`uname -r`
 	realsys="$sys"
-	if [ "$sys" = "AIX" -a -x /usr/bin/oslevel -a -x /usr/sbin/lsattr ]
+	if [ "$sys" = "AIX" -a -x /usr/bin/oslevel -a -x /usr/bin/lsattr ]
 	then
 		rel=`/usr/bin/oslevel`
-		proc=`/usr/sbin/lsdev -C -c processor -S available -F name | head -1`
-		cpu=`/usr/sbin/lsattr -F value -l $proc -a type | sed 's/_.*//'`
+		proc=`/usr/bin/lsdev -C -c processor -S available -F name | head -1`
+		cpu=`/usr/bin/lsattr -F value -l $proc -a type | sed 's/_.*//'`
 	else
 		cpu=`uname -m | sed 's/^i.86$/i386/;s/^sun4u$/sparc64/;s/^i86pc$/i386/;s/^BePC$/i386/;s/^BeMac$/powerpc/;s/^BeBox$/powerpc/'`
 	fi
--- pciutils.spec
+++ pciutils.spec
@@ -35,7 +35,7 @@
 %files
 %defattr(0644, root, root, 0755)
 %attr(0644, root, man) %{_mandir}/man8/*
-%attr(0711, root, root) /usr/sbin/*
+%attr(0711, root, root) /usr/bin/*
 %config /usr/share/pci.ids
 %doc README ChangeLog pciutils.lsm
 
