--- client/scripts/linux
+++ client/scripts/linux
@@ -30,7 +30,7 @@
 # 'ip' just looks too weird. Also, we now have unit-tests! Those unit-tests
 # overwirte this line to use a fake ip-echo tool. It's also convenient
 # if your system holds ip tool in a non-standard location.
-ip=/sbin/ip
+ip=/bin/ip
 
 # update /etc/resolv.conf based on received values
 # This updated version mostly follows Debian script by Andrew Pollock et al.
--- includes/dhcpd.h
+++ includes/dhcpd.h
@@ -1579,7 +1579,7 @@
 #endif
 
 #ifndef _PATH_DHCLIENT_SCRIPT
-#define _PATH_DHCLIENT_SCRIPT	"/sbin/dhclient-script"
+#define _PATH_DHCLIENT_SCRIPT	"/bin/dhclient-script"
 #endif
 
 #ifndef _PATH_DHCLIENT_PID
