--- Makefile
+++ Makefile
@@ -135,10 +135,6 @@
 
 # library
 
-libxxhash.a: ARFLAGS = rcs
-libxxhash.a: xxhash.o
-	$(AR) $(ARFLAGS) $@ $^
-
 $(LIBXXH): LDFLAGS += -shared
 ifeq (,$(filter Windows%,$(OS)))
 $(LIBXXH): CFLAGS += -fPIC
@@ -157,7 +153,7 @@
 
 .PHONY: lib
 lib:  ## generate static and dynamic xxhash libraries
-lib: libxxhash.a libxxhash
+lib: libxxhash
 
 # helper targets
 
@@ -572,11 +568,6 @@
           $< > $@
 
 
-install_libxxhash.a: libxxhash.a
-	@echo Installing libxxhash.a
-	$(Q)$(INSTALL_DIR) $(DESTDIR)$(LIBDIR)
-	$(Q)$(INSTALL_DATA) libxxhash.a $(DESTDIR)$(LIBDIR)
-
 install_libxxhash: libxxhash
 	@echo Installing libxxhash
 	$(Q)$(INSTALL_DIR) $(DESTDIR)$(LIBDIR)
@@ -614,12 +605,11 @@
 	$(Q)ln -sf xxhsum.1 $(DESTDIR)$(MANDIR)/xxh128sum.1
 
 .PHONY: install
-install: install_libxxhash.a install_libxxhash install_libxxhash.includes install_libxxhash.pc install_xxhsum install_man ## install libraries, CLI, links and man page
+install: install_libxxhash install_libxxhash.includes install_libxxhash.pc install_xxhsum install_man ## install libraries, CLI, links and man page
 	@echo xxhash installation completed
 
 .PHONY: uninstall
 uninstall:  ## uninstall libraries, CLI, links and man page
-	$(Q)$(RM) $(DESTDIR)$(LIBDIR)/libxxhash.a
 	$(Q)$(RM) $(DESTDIR)$(LIBDIR)/libxxhash.$(SHARED_EXT)
 	$(Q)$(RM) $(DESTDIR)$(LIBDIR)/libxxhash.$(SHARED_EXT_MAJOR)
 	$(Q)$(RM) $(DESTDIR)$(LIBDIR)/$(LIBXXH)
