--- configure
+++ configure
@@ -132,7 +132,7 @@
 old_cflags="$CFLAGS"
 OBJC='$(OBJZ)'
 PIC_OBJC='$(PIC_OBJZ)'
-INSTALLTARGETS="install-shared install-static"
+INSTALLTARGETS="install-shared"
 UNINSTALLTARGETS="uninstall-shared uninstall-static"
 
 TEST="teststatic"
@@ -622,7 +622,7 @@
   UNINSTALLTARGETS=uninstall-static
   echo Building static library $STATICLIB version $VER with $CC. | tee -a configure.log
 else
-  ALL="static shared"
+  ALL="shared"
   TEST="${TEST} testshared"
 fi
 
