--- Makefile.in
+++ Makefile.in
@@ -2617,11 +2617,11 @@
 DISTCLEANFILES = tool-versions.txt
 
 # "gnulib-tests" is the gnulib unit test dir.
-SUBDIRS = gl build-aux lib find xargs locate doc po m4 gnulib-tests
+SUBDIRS = gl build-aux lib find xargs doc po m4 gnulib-tests
 ALL_RECURSIVE_TARGETS = check-root
 ACLOCAL_AMFLAGS = -I gl/m4 -I m4
 TESTFILE_SUFFIXES = .exp .xo .xe .xi
-built_programs = find xargs frcode locate updatedb
+built_programs = find xargs
 
 # Indirections required so that we'll still be able to know the
 # complete list of our tests even if the user overrides TESTS
