--- etc/login.defs
+++ etc/login.defs
@@ -127,7 +127,7 @@
 #   Directory where mailboxes reside, _or_ name of file, relative to the
 #   home directory.  If you _do_ define both, MAIL_DIR takes precedence.
 #
-MAIL_DIR	/var/spool/mail
+MAIL_DIR	/var/mail
 #MAIL_FILE	.mail
 
 #
@@ -329,7 +329,7 @@
 # Note: If you use PAM, it is recommended to use a value consistent with
 # the PAM modules configuration.
 #
-#ENCRYPT_METHOD DES
+ENCRYPT_METHOD YESCRYPT
 
 #
 # Only works if ENCRYPT_METHOD is set to SHA256 or SHA512.
@@ -374,7 +374,7 @@
 # If not specified, a cost factor of 5 will be used.
 # The value must be within the 1-11 range.
 #
-#YESCRYPT_COST_FACTOR 5
+YESCRYPT_COST_FACTOR 11
 
 #
 # List of groups to add to the user's supplementary group set
