--- share/initialization/default
+++ share/initialization/default
@@ -3,25 +3,7 @@
 # initialization script cannot be read from $XDG_CONFIG_HOME/yash/rc or ~/.yashrc.
 # This file is in the public domain.
 
-# print introduction
-(
 command . --autoload config 2>/dev/null || yashdatadir='?'
-cat <<__END__
-Welcome to yash!
-
-You are seeing this message because a user-specific initialization script is
-not found at ${XDG_CONFIG_HOME:-$HOME/.config}/yash/rc or ~/.yashrc.
-The current session has been loaded with common
-settings that should be useful for many users. To suppress this message, copy
-a sample initialization script from ${yashdatadir}/initialization/sample
-to ~/.yashrc and make any customization in it if you want. For bare default
-settings, put an empty file at ~/.yashrc. Without ~/.yashrc, you will see this
-message again next time you start yash.
-
-For more instructions, see the manual at: https://magicant.github.io/yash/doc/
-
-__END__
-)
 
 # load a sample initialization script
 . --autoload initialization/sample
--- share/initialization/sample
+++ share/initialization/sample
@@ -10,15 +10,15 @@
 
 # These are additional aliases that are not defined in the common script.
 # Uncomment to enable them.
-#alias g='grep'
-#alias l='$PAGER'
+alias grep='grep --color=auto'
+alias ls='ls -F --color=auto'
 #alias --global L='|$PAGER'
 #alias --global N='>/dev/null 2>&1' N1='>/dev/null' N2='2>/dev/null'
 
 # Uncomment if you want to clear the screen with Ctrl-L.
-#bindkey --emacs '\^L' clear-and-redraw-all
-#bindkey --vi-insert '\^L' clear-and-redraw-all
-#bindkey --vi-command '\^L' clear-and-redraw-all
+bindkey --emacs '\^L' clear-and-redraw-all
+bindkey --vi-insert '\^L' clear-and-redraw-all
+bindkey --vi-command '\^L' clear-and-redraw-all
 
 # Uncomment to enable direnv support. (jq required)
 #_update_direnv() {
@@ -37,6 +37,10 @@
 
 # And add your own customization below.
 
+# Use "Ctrl + Right Arrow" to move the cursor to the next word
+bindkey --emacs '\^[[1;5C' forward-emacsword
 
+# Use "Ctrl + Left Arrow" to move the cursor to the previous word
+bindkey --emacs '\^[[1;5D' backward-emacsword
 
 # vim: set et sw=2 sts=2 tw=78 ft=sh:
