--- share/initialization/sample
+++ share/initialization/sample
@@ -2,15 +2,14 @@
 # This is a sample initialization script for yash. Copy this file to ~/.yashrc
 # and add your favorite customization to it.
 
-
 # Firstly, load the common customization script.
 # If you don't like settings applied in this script, remove this line.
 . --autoload --no-alias initialization/common
 
 # These are additional aliases that are not defined in the common script.
 # Uncomment to enable them.
-#alias g='grep'
-#alias l='$PAGER'
+alias grep='grep --color=auto'
+alias ls='ls -F --color=auto'
 #alias --global L='|$PAGER'
 #alias --global N='>/dev/null 2>&1' N1='>/dev/null' N2='2>/dev/null'
 
@@ -31,6 +30,12 @@
 
 # And add your own customization below.
 
-
+if [ -f /etc/profile ]; then
+  . /etc/profile
+fi
+
+if [ -f /etc/yashrc ]; then
+  . /etc/yashrc
+fi
 
 # vim: set et sw=2 sts=2 tw=78 ft=sh:
