--- include/pathnames.h
+++ include/pathnames.h
@@ -67,9 +67,12 @@
 # define _PATH_TMP		"/tmp/"
 #endif
 
-#ifndef _PATH_BTMP
-# define _PATH_BTMP		"/var/log/btmp"
-#endif
+#undef _PATH_UTMP
+#define _PATH_UTMP		UTMPX_FILE
+#undef _PATH_WTMP
+#define _PATH_WTMP		WTMPX_FILE
+#undef _PATH_BTMP
+#define _PATH_BTMP		"/var/log/btmp"
 
 #define _PATH_ISSUE_FILENAME	"issue"
 #define _PATH_ISSUE_DIRNAME	_PATH_ISSUE_FILENAME ".d"
--- login-utils/login.c
+++ login-utils/login.c
@@ -72,6 +72,9 @@
 # include <libaudit.h>
 #endif
 
+#include <utmps/config.h>
+#include <utmps/utmps.h>
+
 #include "c.h"
 #include "pathnames.h"
 #include "strutils.h"
@@ -630,7 +633,7 @@
 			       sizeof(ut.ut_addr_v6));
 	}
 
-	updwtmpx(_PATH_BTMP, &ut);
+	utmps_updwtmpx_g("/run/utmps/.btmpd-socket", &ut, 0);
 }
 
 #ifdef HAVE_LIBAUDIT
@@ -747,7 +750,7 @@
 
 	/* Find pid in utmp.
 	 *
-	 * login sometimes overwrites the runlevel entry in /var/run/utmp,
+	 * login sometimes overwrites the runlevel entry in /run/utmps/utmp,
 	 * confusing sysvinit. I added a test for the entry type, and the
 	 * problem was gone. (In a runlevel entry, st_pid is not really a pid
 	 * but some number calculated from the previous and current runlevel.)
@@ -805,7 +808,7 @@
 	pututxline(&ut);
 	endutxent();
 
-	updwtmpx(_PATH_WTMP, &ut);
+	utmps_updwtmpx_g("/run/utmps/.wtmpd-socket", &ut, 0);
 }
 
 static void log_syslog(struct login_context *cxt)
