--- man/Makefile.in
+++ man/Makefile.in
@@ -393,7 +393,7 @@
 man_MANS = man1/chage.1 man1/chfn.1 man8/chgpasswd.8 man8/chpasswd.8 \
 	man1/chsh.1 man1/expiry.1 man5/faillog.5 man8/faillog.8 \
 	man3/getspnam.3 man1/gpasswd.1 man8/groupadd.8 man8/groupdel.8 \
-	man8/groupmems.8 man8/groupmod.8 man1/groups.1 man8/grpck.8 \
+	man8/groupmems.8 man8/groupmod.8 man1/ man8/grpck.8 \
 	man8/grpconv.8 man8/grpunconv.8 man5/gshadow.5 man8/lastlog.8 \
 	man1/login.1 man5/login.defs.5 man8/logoutd.8 man1/newgrp.1 \
 	man8/newusers.8 man8/nologin.8 man1/passwd.1 man5/passwd.5 \
--- man/cs/Makefile.in
+++ man/cs/Makefile.in
@@ -326,7 +326,7 @@
 	man8/groupadd.8 \
 	man8/groupdel.8 \
 	man8/groupmod.8 \
-	man1/groups.1 \
+	man1/ \
 	man8/grpck.8 \
 	man5/gshadow.5 \
 	man8/lastlog.8 \
--- man/da/Makefile.in
+++ man/da/Makefile.in
@@ -323,7 +323,7 @@
 top_srcdir = @top_srcdir@
 
 # 2012.01.28 - activate manpages with more than 50% translated messages
-man_MANS = man1/chfn.1 man8/groupdel.8 man1/groups.1 man5/gshadow.5 \
+man_MANS = man1/chfn.1 man8/groupdel.8 man1/ man5/gshadow.5 \
 	man8/logoutd.8 man1/newgrp.1 man8/nologin.8 man1/sg.1 \
 	man8/vigr.8 man8/vipw.8 $(am__append_1)
 man_nopam = 
--- man/de/Makefile.in
+++ man/de/Makefile.in
@@ -325,7 +325,7 @@
 man_MANS = man1/chage.1 man1/chfn.1 man8/chgpasswd.8 man8/chpasswd.8 \
 	man1/chsh.1 man1/expiry.1 man5/faillog.5 man8/faillog.8 \
 	man3/getspnam.3 man1/gpasswd.1 man8/groupadd.8 man8/groupdel.8 \
-	man8/groupmems.8 man8/groupmod.8 man1/groups.1 man8/grpck.8 \
+	man8/groupmems.8 man8/groupmod.8 man1/ man8/grpck.8 \
 	man8/grpconv.8 man8/grpunconv.8 man5/gshadow.5 man8/lastlog.8 \
 	man1/login.1 man5/login.defs.5 man8/logoutd.8 man1/newgrp.1 \
 	man8/newusers.8 man8/nologin.8 man1/passwd.1 man5/passwd.5 \
--- man/fr/Makefile.in
+++ man/fr/Makefile.in
@@ -327,7 +327,7 @@
 man_MANS = man1/chage.1 man1/chfn.1 man8/chgpasswd.8 man8/chpasswd.8 \
 	man1/chsh.1 man1/expiry.1 man5/faillog.5 man8/faillog.8 \
 	man3/getspnam.3 man1/gpasswd.1 man8/groupadd.8 man8/groupdel.8 \
-	man8/groupmems.8 man8/groupmod.8 man1/groups.1 man8/grpck.8 \
+	man8/groupmems.8 man8/groupmod.8 man1/ man8/grpck.8 \
 	man8/grpconv.8 man8/grpunconv.8 man5/gshadow.5 man8/lastlog.8 \
 	man1/login.1 man5/login.defs.5 man8/logoutd.8 man1/newgrp.1 \
 	man8/newusers.8 man8/nologin.8 man1/passwd.1 man5/passwd.5 \
--- man/hu/Makefile.in
+++ man/hu/Makefile.in
@@ -321,7 +321,7 @@
 man_MANS = \
 	man1/chsh.1 \
 	man1/gpasswd.1 \
-	man1/groups.1 \
+	man1/ \
 	man8/lastlog.8 \
 	man1/login.1 \
 	man1/newgrp.1 \
--- man/it/Makefile.in
+++ man/it/Makefile.in
@@ -325,7 +325,7 @@
 man_MANS = man1/chage.1 man1/chfn.1 man8/chgpasswd.8 man8/chpasswd.8 \
 	man1/chsh.1 man1/expiry.1 man5/faillog.5 man8/faillog.8 \
 	man3/getspnam.3 man1/gpasswd.1 man8/groupadd.8 man8/groupdel.8 \
-	man8/groupmems.8 man8/groupmod.8 man1/groups.1 man8/grpck.8 \
+	man8/groupmems.8 man8/groupmod.8 man1/ man8/grpck.8 \
 	man8/grpconv.8 man8/grpunconv.8 man5/gshadow.5 man8/lastlog.8 \
 	man1/login.1 man5/login.defs.5 man8/logoutd.8 man1/newgrp.1 \
 	man8/newusers.8 man8/nologin.8 man1/passwd.1 man5/passwd.5 \
--- man/ja/Makefile.in
+++ man/ja/Makefile.in
@@ -322,7 +322,7 @@
 top_srcdir = @top_srcdir@
 man_MANS = man1/chage.1 man1/chfn.1 man8/chpasswd.8 man1/chsh.1 \
 	man1/expiry.1 man5/faillog.5 man8/faillog.8 man1/gpasswd.1 \
-	man8/groupadd.8 man8/groupdel.8 man8/groupmod.8 man1/groups.1 \
+	man8/groupadd.8 man8/groupdel.8 man8/groupmod.8 man1/ \
 	man8/grpck.8 man8/grpconv.8 man8/grpunconv.8 man8/lastlog.8 \
 	man1/login.1 man5/login.defs.5 man8/logoutd.8 man1/newgrp.1 \
 	man8/newusers.8 man1/passwd.1 man5/passwd.5 man8/pwck.8 \
--- man/ko/Makefile.in
+++ man/ko/Makefile.in
@@ -321,7 +321,7 @@
 man_MANS = \
 	man1/chfn.1 \
 	man1/chsh.1 \
-	man1/groups.1 \
+	man1/ \
 	man1/login.1 \
 	man5/passwd.5 \
 	man1/su.1 \
--- man/pl/Makefile.in
+++ man/pl/Makefile.in
@@ -326,7 +326,7 @@
 # 2012.01.28 - activate manpages with more than 50% translated messages
 man_MANS = man1/chage.1 man1/chsh.1 man1/expiry.1 man5/faillog.5 \
 	man8/faillog.8 man3/getspnam.3 man8/groupadd.8 man8/groupdel.8 \
-	man8/groupmems.8 man8/groupmod.8 man1/groups.1 man8/grpck.8 \
+	man8/groupmems.8 man8/groupmod.8 man1/ man8/grpck.8 \
 	man8/lastlog.8 man8/logoutd.8 man1/newgrp.1 man1/sg.1 \
 	man3/shadow.3 man8/userdel.8 man8/usermod.8 man8/vigr.8 \
 	man8/vipw.8 $(am__append_1)
--- man/ru/Makefile.in
+++ man/ru/Makefile.in
@@ -325,7 +325,7 @@
 man_MANS = man1/chage.1 man1/chfn.1 man8/chgpasswd.8 man8/chpasswd.8 \
 	man1/chsh.1 man1/expiry.1 man5/faillog.5 man8/faillog.8 \
 	man3/getspnam.3 man1/gpasswd.1 man8/groupadd.8 man8/groupdel.8 \
-	man8/groupmems.8 man8/groupmod.8 man1/groups.1 man8/grpck.8 \
+	man8/groupmems.8 man8/groupmod.8 man1/ man8/grpck.8 \
 	man8/grpconv.8 man8/grpunconv.8 man5/gshadow.5 man8/lastlog.8 \
 	man1/login.1 man5/login.defs.5 man8/logoutd.8 man1/newgrp.1 \
 	man8/newusers.8 man8/nologin.8 man1/passwd.1 man5/passwd.5 \
--- man/sv/Makefile.in
+++ man/sv/Makefile.in
@@ -326,7 +326,7 @@
 # 2012.01.28 - activate manpages with more than 50% translated messages
 man_MANS = man1/chage.1 man1/chsh.1 man1/expiry.1 man5/faillog.5 \
 	man8/faillog.8 man3/getspnam.3 man8/groupadd.8 man8/groupdel.8 \
-	man8/groupmems.8 man8/groupmod.8 man1/groups.1 man8/grpck.8 \
+	man8/groupmems.8 man8/groupmod.8 man1/ man8/grpck.8 \
 	man5/gshadow.5 man8/lastlog.8 man8/logoutd.8 man1/newgrp.1 \
 	man8/nologin.8 man1/passwd.1 man5/passwd.5 man8/pwck.8 \
 	man1/sg.1 man3/shadow.3 man5/suauth.5 man8/userdel.8 \
--- man/zh_CN/Makefile.in
+++ man/zh_CN/Makefile.in
@@ -325,7 +325,7 @@
 man_MANS = man1/chage.1 man1/chfn.1 man8/chgpasswd.8 man8/chpasswd.8 \
 	man1/chsh.1 man1/expiry.1 man5/faillog.5 man8/faillog.8 \
 	man3/getspnam.3 man1/gpasswd.1 man8/groupadd.8 man8/groupdel.8 \
-	man8/groupmems.8 man8/groupmod.8 man1/groups.1 man8/grpck.8 \
+	man8/groupmems.8 man8/groupmod.8 man1/ man8/grpck.8 \
 	man8/grpconv.8 man8/grpunconv.8 man5/gshadow.5 man8/lastlog.8 \
 	man1/login.1 man5/login.defs.5 man8/logoutd.8 man1/newgrp.1 \
 	man8/newusers.8 man8/nologin.8 man1/passwd.1 man5/passwd.5 \
--- src/Makefile.in
+++ src/Makefile.in
@@ -88,8 +88,7 @@
 POST_UNINSTALL = :
 build_triplet = @build@
 host_triplet = @host@
-bin_PROGRAMS = groups$(EXEEXT) login$(EXEEXT) $(am__EXEEXT_1) \
-	$(am__EXEEXT_2)
+bin_PROGRAMS = login$(EXEEXT) $(am__EXEEXT_1) $(am__EXEEXT_2)
 sbin_PROGRAMS = nologin$(EXEEXT)
 ubin_PROGRAMS = faillog$(EXEEXT) lastlog$(EXEEXT) chage$(EXEEXT) \
 	chfn$(EXEEXT) chsh$(EXEEXT) expiry$(EXEEXT) gpasswd$(EXEEXT) \
