--- library/uptime.c
+++ library/uptime.c
@@ -30,7 +30,7 @@
 #include <string.h>
 #include <time.h>
 #include <unistd.h>
-#include <utmp.h>
+#include <utmpx.h>
 #ifdef WITH_SYSTEMD
 #include <systemd/sd-daemon.h>
 #include <systemd/sd-login.h>
@@ -51,19 +51,19 @@
 static int count_users(void)
 {
     int numuser = 0;
-    struct utmp *ut;
+    struct utmpx *ut;
 
 #if defined(WITH_SYSTEMD) || defined(WITH_ELOGIND)
     if (sd_booted() > 0)
       return sd_get_sessions(NULL);
 #endif
 
-    setutent();
-    while ((ut = getutent())) {
+    setutxent();
+    while ((ut = getutxent())) {
     if ((ut->ut_type == USER_PROCESS) && (ut->ut_name[0] != '\0'))
         numuser++;
     }
-    endutent();
+    endutxent();
 
     return numuser;
 }
--- src/w.c
+++ src/w.c
@@ -44,14 +44,7 @@
 #include <termios.h>
 #include <time.h>
 #include <unistd.h>
-#ifdef HAVE_UTMPX_H
 #include <utmpx.h>
-#ifndef HAVE_UT_HOSTSIZE_IN_UTMPX
-#include <utmp.h>
-#endif
-#else
-#	include <utmp.h>
-#endif
 #include <arpa/inet.h>
 #ifdef WITH_SYSTEMD
 #      include <systemd/sd-login.h>
@@ -72,11 +65,7 @@
 static int ignoreuser = 0;	/* for '-u' */
 static int oldstyle = 0;	/* for '-o' */
 
-#ifdef HAVE_UTMPX_H
 typedef struct utmpx utmp_t;
-#else
-typedef struct utmp utmp_t;
-#endif
 
 #ifdef __GLIBC__
 #if !defined(UT_HOSTSIZE) || defined(__UT_HOSTSIZE)
@@ -595,11 +584,7 @@
 #endif
 			   u, ip_addresses, fromlen);
 
-#ifdef HAVE_UTMPX_H
             print_logintime(u->ut_tv.tv_sec, stdout);
-#else
-            print_logintime(u->ut_time, stdout);
-#endif
 #if (defined(WITH_SYSTEMD) || defined(WITH_ELOGIND)) && defined(HAVE_SD_SESSION_GET_LEADER)
         }
 #endif
@@ -838,19 +823,10 @@
 		}
 	} else {
 #endif
-#ifdef HAVE_UTMPX_H
 	setutxent();
-#else
-	utmpname(UTMP_FILE);
-	setutent();
-#endif
 	if (user) {
 		for (;;) {
-#ifdef HAVE_UTMPX_H
 			u = getutxent();
-#else
-			u = getutent();
-#endif
 			if (!u)
 				break;
 			if (u->ut_type != USER_PROCESS)
@@ -865,11 +841,7 @@
 		}
 	} else {
 		for (;;) {
-#ifdef HAVE_UTMPX_H
 			u = getutxent();
-#else
-			u = getutent();
-#endif
 			if (!u)
 				break;
 			if (u->ut_type != USER_PROCESS)
@@ -883,11 +855,7 @@
 					 fromlen, ip_addresses, pids);
 		}
 	}
-#ifdef HAVE_UTMPX_H
 	endutxent();
-#else
-	endutent();
-#endif
 #if (defined(WITH_SYSTEMD) || defined(WITH_ELOGIND)) && defined(HAVE_SD_SESSION_GET_LEADER)
 	}
 #endif
