--- proc/uptime.c
+++ proc/uptime.c
@@ -29,7 +29,7 @@
 #include <string.h>
 #include <time.h>
 #include <unistd.h>
-#include <utmp.h>
+#include <utmpx.h>
 
 #include <proc/misc.h>
 #include "procps-private.h"
@@ -42,14 +42,14 @@
 static int count_users(void)
 {
     int numuser = 0;
-    struct utmp *ut;
+    struct utmpx *u;
 
-    setutent();
-    while ((ut = getutent())) {
-    if ((ut->ut_type == USER_PROCESS) && (ut->ut_name[0] != '\0'))
+    setutxent();
+    while ((u = getutxent())) {
+    if ((u->ut_type == USER_PROCESS) && (u->ut_name[0] != '\0'))
         numuser++;
     }
-    endutent();
+    endutxent();
 
     return numuser;
 }
--- w.c
+++ w.c
@@ -43,11 +43,7 @@
 #include <termios.h>
 #include <time.h>
 #include <unistd.h>
-#ifdef HAVE_UTMPX_H
-#	include <utmpx.h>
-#else
-#	include <utmp.h>
-#endif
+#include <utmpx.h>
 #include <arpa/inet.h>
 
 #include "c.h"
@@ -60,11 +56,7 @@
 static int ignoreuser = 0;	/* for '-u' */
 static int oldstyle = 0;	/* for '-o' */
 
-#ifdef HAVE_UTMPX_H
 typedef struct utmpx utmp_t;
-#else
-typedef struct utmp utmp_t;
-#endif
 
 #if !defined(UT_HOSTSIZE) || defined(__UT_HOSTSIZE)
 #	define UT_HOSTSIZE __UT_HOSTSIZE
@@ -484,11 +476,7 @@
         printf("%-*.*s%-9.8s", userlen + 1, userlen, uname, u->ut_line);
         if (from)
             print_from(u, ip_addresses, fromlen);
-#ifdef HAVE_UTMPX_H
         print_logintime(u->ut_tv.tv_sec, stdout);
-#else
-        print_logintime(u->ut_time, stdout);
-#endif
         if (*u->ut_line == ':')
             /* idle unknown for xdm logins */
             printf(" ?xdm? ");
@@ -667,19 +655,10 @@
 			printf(_("   IDLE WHAT\n"));
 	}
 
-#ifdef HAVE_UTMPX_H
 	setutxent();
-#else
-	utmpname(UTMP_FILE);
-	setutent();
-#endif
 	if (user) {
 		for (;;) {
-#ifdef HAVE_UTMPX_H
 			u = getutxent();
-#else
-			u = getutent();
-#endif
 			if (!u)
 				break;
 			if (u->ut_type != USER_PROCESS)
@@ -690,11 +669,7 @@
 		}
 	} else {
 		for (;;) {
-#ifdef HAVE_UTMPX_H
 			u = getutxent();
-#else
-			u = getutent();
-#endif
 			if (!u)
 				break;
 			if (u->ut_type != USER_PROCESS)
@@ -704,11 +679,7 @@
 					 fromlen, ip_addresses);
 		}
 	}
-#ifdef HAVE_UTMPX_H
 	endutxent();
-#else
-	endutent();
-#endif
 
 	return EXIT_SUCCESS;
 }
