--- src/watch.c
+++ src/watch.c
@@ -53,10 +53,8 @@
 # define _XOPEN_SOURCE_EXTENDED 1
 # include <wchar.h>
 # include <wctype.h>
-# include <ncursesw/ncurses.h>
-#else
-# include <ncurses.h>
 #endif	/* WITH_WATCH8BIT */
+#include <ncurses.h>
 
 #ifdef FORCE_8BIT
 # undef isprint
@@ -711,7 +709,7 @@
 #ifdef WITH_WATCH8BIT
 				cchar_t oldc;
 				in_wch(&oldc);
-				exit_early = (wchar_t) c != oldc.chars[0];
+				exit_early = (wchar_t) c != oldc.vals[0];
 #else
 				chtype oldch = inch();
 				unsigned char oldc = oldch & A_CHARTEXT;
@@ -723,7 +721,7 @@
 #ifdef WITH_WATCH8BIT
 				cchar_t oldc;
 				in_wch(&oldc);
-				if ((wchar_t) c == oldc.chars[0])
+				if ((wchar_t) c == oldc.vals[0])
 					unchanged_buffer++;
 #else
 				chtype oldch = inch();
@@ -737,10 +735,10 @@
 				cchar_t oldc;
 				in_wch(&oldc);
 				attr = !first_screen
-				    && ((wchar_t) c != oldc.chars[0]
+				    && ((wchar_t) c != oldc.vals[0]
 					||
 					((flags & WATCH_CUMUL)
-					 && (oldc.attr & A_ATTRIBUTES)));
+					 && (oldc.attributes & A_ATTRIBUTES)));
 #else
 				chtype oldch = inch();
 				unsigned char oldc = oldch & A_CHARTEXT;
