--- Makefile
+++ Makefile
@@ -17,14 +17,14 @@
 
 install: hostname
 	install -d ${BASEDIR}$(MANDIR)/man1
-	install -o root -g root -m 0644 hostname.1 ${BASEDIR}$(MANDIR)/man1
+	install -m 0644 hostname.1 ${BASEDIR}$(MANDIR)/man1
 	ln -fs hostname.1 ${BASEDIR}$(MANDIR)/man1/dnsdomainname.1
 	ln -fs hostname.1 ${BASEDIR}$(MANDIR)/man1/domainname.1
 	ln -fs hostname.1 ${BASEDIR}$(MANDIR)/man1/ypdomainname.1
 	ln -fs hostname.1 ${BASEDIR}$(MANDIR)/man1/nisdomainname.1
 
 	install -d ${BASEDIR}$(BINDIR)
-	install -o root -g root -m 0755 hostname ${BASEDIR}$(BINDIR)
+	install -m 0755 hostname ${BASEDIR}$(BINDIR)
 	ln -fs hostname ${BASEDIR}$(BINDIR)/dnsdomainname
 	ln -fs hostname ${BASEDIR}$(BINDIR)/domainname
 	ln -fs hostname ${BASEDIR}$(BINDIR)/nisdomainname
