--- e2fsck/unix.c
+++ e2fsck/unix.c
@@ -604,7 +604,7 @@
 	return 0;
 }
 
-#define PATH_SET "PATH=/sbin"
+#define PATH_SET "PATH=/bin"
 
 /*
  * Make sure 0,1,2 file descriptors are open, so that we don't open
@@ -1125,7 +1125,7 @@
 	sigaction(SIGUSR2, &sa, 0);
 #endif
 
-	/* Update our PATH to include /sbin if we need to run badblocks  */
+	/* Update our PATH to include /bin if we need to run badblocks  */
 	if (cflag) {
 		char *oldpath = getenv("PATH");
 		char *newpath;
--- lib/uuid/uuidd.h
+++ lib/uuid/uuidd.h
@@ -38,7 +38,7 @@
 #define UUIDD_DIR		"/var/lib/libuuid"
 #define UUIDD_SOCKET_PATH	UUIDD_DIR "/request"
 #define UUIDD_PIDFILE_PATH	UUIDD_DIR "/uuidd.pid"
-#define UUIDD_PATH		"/usr/sbin/uuidd"
+#define UUIDD_PATH		"/usr/bin/uuidd"
 
 #define UUIDD_OP_GETPID			0
 #define UUIDD_OP_GET_MAXOP		1
--- misc/fsck.c
+++ misc/fsck.c
@@ -120,7 +120,7 @@
 static char *fstype = NULL;
 static struct fs_info *filesys_info = NULL, *filesys_last = NULL;
 static struct fsck_instance *instance_list;
-static const char *fsck_prefix_path = "/sbin:/sbin/fs.d:/sbin/fs:/etc/fs:/etc";
+static const char *fsck_prefix_path = "/bin:/bin/fs.d:/bin/fs:/etc/fs:/etc";
 static char *fsck_path = 0;
 static blkid_cache cache = NULL;
 
--- misc/mke2fs.c
+++ misc/mke2fs.c
@@ -792,7 +792,7 @@
 	return 1;
 }
 
-#define PATH_SET "PATH=/sbin"
+#define PATH_SET "PATH=/bin"
 
 static void parse_extended_opts(struct ext2_super_block *param,
 				const char *opts)
@@ -1593,7 +1593,7 @@
 	}
 	strcpy(newpath, PATH_SET);
 
-	/* Update our PATH to include /sbin  */
+	/* Update our PATH to include /bin  */
 	if (oldpath) {
 		strcat (newpath, ":");
 		strcat (newpath, oldpath);
--- scrub/e2scrub.in
+++ scrub/e2scrub.in
@@ -23,7 +23,7 @@
 # check filesystems in VGs that have at least 256MB (or so) of
 # free space.
 
-PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
+PATH=/usr/bin:/bin
 
 if (( $EUID != 0 )); then
     echo "e2scrub must be run as root"
--- scrub/e2scrub_all.in
+++ scrub/e2scrub_all.in
@@ -18,7 +18,7 @@
 #  along with this program; if not, write the Free Software Foundation,
 #  Inc.,  51 Franklin St, Fifth Floor, Boston, MA  02110-1301, USA.
 
-PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
+PATH=/usr/bin:/bin
 
 if (( $EUID != 0 )); then
     echo "e2scrub_all must be run as root"
