--- config.mk
+++ config.mk
@@ -1,12 +1,12 @@
 # sdhcp version
 VERSION   = 0.1
 
-PREFIX    = /usr/local
-DESTDIR   =
-MANPREFIX = $(PREFIX)/share/man
+PREFIX    = /usr
+DESTDIR   = $(DESTDIR)
+MANPREFIX = /usr/share/man
 
-CC        = cc
+CC        = $(CC)
 LD        = $(CC)
 CPPFLAGS  = -D_DEFAULT_SOURCE
-CFLAGS    = -Wall -Wextra -pedantic -std=c99 $(CPPFLAGS)
-LDFLAGS   = -s
+CFLAGS    = -Wall -Wextra -pedantic -std=c99 $(CFLAGS) $(CPPFLAGS)
+LDFLAGS   = $(LDFLAGS)
