--- Makefile
+++ Makefile
@@ -581,7 +581,7 @@
 
 SHELL=		/bin/sh
 
-all:		tzselect zic zdump libtz.a $(TABDATA) \
+all:		tzselect zic zdump $(TABDATA) \
 		  vanguard.zi main.zi rearguard.zi
 
 ALL:		all date $(ENCHILADA)
@@ -600,8 +600,6 @@
 		cp tzselect '$(DESTDIR)$(BINDIR)/.'
 		cp zdump '$(DESTDIR)$(ZDUMPDIR)/.'
 		cp zic '$(DESTDIR)$(ZICDIR)/.'
-		cp libtz.a '$(DESTDIR)$(LIBDIR)/.'
-		$(RANLIB) '$(DESTDIR)$(LIBDIR)/libtz.a'
 		cp -f newctime.3 newtzset.3 '$(DESTDIR)$(MANDIR)/man3/.'
 		cp -f tzfile.5 '$(DESTDIR)$(MANDIR)/man5/.'
 		cp -f tzselect.8 zdump.8 zic.8 '$(DESTDIR)$(MANDIR)/man8/.'
@@ -753,11 +751,6 @@
 force_tzs:	$(TZS_NEW)
 		cp $(TZS_NEW) $(TZS)
 
-libtz.a:	$(LIBOBJS)
-		rm -f $@
-		$(AR) -rc $@ $(LIBOBJS)
-		$(RANLIB) $@
-
 date:		$(DATEOBJS)
 		$(CC) -o $@ $(CFLAGS) $(LDFLAGS) $(DATEOBJS) $(LDLIBS)
 
@@ -902,7 +895,7 @@
 		rm -fr check_*.dir
 		rm -f *.o *.out $(TIME_T_ALTERNATIVES) \
 		  check_* core typecheck_* \
-		  date tzselect version.h zdump zic libtz.a
+		  date tzselect version.h zdump zic
 clean:		clean_misc
 		rm -fr *.dir tzdb-*/
 		rm -f *.zi $(TZS_NEW)
