--- Makefile
+++ Makefile
@@ -629,7 +629,7 @@
 		ziguard.awk zishrink.awk \
 		zone.tab zone1970.tab zonenow.tab
 
-all:		tzselect zic zdump libtz.a $(TABDATA) \
+all:		tzselect zic zdump $(TABDATA) \
 		  vanguard.zi main.zi rearguard.zi
 
 ALL:		all date $(ENCHILADA)
@@ -648,8 +648,6 @@
 		cp tzselect '$(DESTDIR)$(BINDIR)/.'
 		cp zdump '$(DESTDIR)$(ZDUMPDIR)/.'
 		cp zic '$(DESTDIR)$(ZICDIR)/.'
-		cp libtz.a '$(DESTDIR)$(LIBDIR)/.'
-		$(RANLIB) '$(DESTDIR)$(LIBDIR)/libtz.a'
 		cp -f newctime.3 newtzset.3 '$(DESTDIR)$(MANDIR)/man3/.'
 		cp -f tzfile.5 '$(DESTDIR)$(MANDIR)/man5/.'
 		cp -f tzselect.8 zdump.8 zic.8 '$(DESTDIR)$(MANDIR)/man8/.'
@@ -833,11 +831,6 @@
 force_tzs:	$(TZS_NEW)
 		cp $(TZS_NEW) $(TZS)
 
-libtz.a:	$(LIBOBJS)
-		rm -f $@
-		$(AR) $(ARFLAGS) $@ $(LIBOBJS)
-		$(RANLIB) $@
-
 date:		$(DATEOBJS)
 		$(CC) -o $@ $(CFLAGS) $(LDFLAGS) $(DATEOBJS) $(LDLIBS)
 
@@ -1014,7 +1007,7 @@
 		rm -fr check_*.dir typecheck_*.dir
 		rm -f *.o *.out $(TIME_T_ALTERNATIVES) \
 		  check_* core typecheck_* \
-		  date tzdir.h tzselect version.h zdump zic libtz.a
+		  date tzdir.h tzselect version.h zdump zic
 clean:		clean_misc
 		rm -fr *.dir tzdb-*/
 		rm -f *.zi $(TZS_NEW)
