--- configure
+++ configure
@@ -35,8 +35,8 @@
 
 SOURCEDIR=`dirname "${0}"`
 
-MANPATH_BASE="/usr/share/man:/usr/X11R6/man"
-MANPATH_DEFAULT="/usr/share/man:/usr/X11R6/man:/usr/local/man"
+MANPATH_BASE=/usr/share/man
+MANPATH_DEFAULT=/usr/share/man
 OSENUM=
 OSNAME=
 UTF8_LOCALE=
@@ -102,7 +102,7 @@
 MANDOC_COBJS=
 SOELIM_COBJS=
 
-PREFIX="/usr/local"
+PREFIX=/usr
 BINDIR=
 SBINDIR=
 BIN_FROM_SBIN=
--- configure.local.example
+++ configure.local.example
@@ -73,12 +73,12 @@
 # available, manuals are searched for in the following directory
 # trees by default.
 
-MANPATH_DEFAULT="/usr/share/man:/usr/X11R6/man:/usr/local/man"
+MANPATH_DEFAULT=/usr/share/man
 
 # Validation of cross references with mandoc -Tlint only looks
 # for manual pages in the following directories:
 
-MANPATH_BASE="/usr/share/man:/usr/X11R6/man"
+MANPATH_BASE=/usr/share/man
 
 # When man(1) is called with the -S option and no manual page is
 # found matching the requested name and the requested architecture,
@@ -109,7 +109,7 @@
 # Even if you set PREFIX to something else, the other variables
 # pick it up without copying them all over.
 
-PREFIX="/usr/local"
+PREFIX=/usr
 BINDIR="${PREFIX}/bin"
 SBINDIR="${PREFIX}/sbin"
 MANDIR="${PREFIX}/man"
@@ -224,7 +224,7 @@
 # of a colon-separated path:
 
 # Homebrow package manager on Mac OS X:
-PREFIX="/usr/local"
+PREFIX=/usr
 READ_ALLOWED_PATH="${PREFIX}/Cellar"
 
 # Nix package manager and/or NixOS Linux distribution:
@@ -234,7 +234,7 @@
 READ_ALLOWED_PATH="/gnu/store"
 
 # If multiple package managers are used concurrently:
-PREFIX="/usr/local"
+PREFIX=/usr
 READ_ALLOWED_PATH="/nix/store:${PREFIX}/Cellar"
 
 # --- user settings for the mandoc(3) library --------------------------
--- gmdiff
+++ gmdiff
@@ -25,7 +25,6 @@
 
 if [ "X$1" = "X-h" ]; then
   shift
-  export PATH="/usr/local/heirloom-doctools/bin:$PATH"
   EQN="neqn"
   ROFF="nroff"
   MOPT="-Ios=BSD -Tascii $MOPT"
--- INSTALL
+++ INSTALL
@@ -43,9 +43,8 @@
 functionality as your system's "catman", if your operating
 system contains one.
 
-3. Define MANPATH_DEFAULT in configure.local
-if /usr/share/man:/usr/X11R6/man:/usr/local/man is not appropriate
-for your operating system.
+3. Define MANPATH_DEFAULT in configure.local if /usr/share/man is not
+appropriate for your operating system.
 
 4. Run "./configure".
 This script attempts autoconfiguration of mandoc for your system.
--- man.1
+++ man.1
@@ -224,7 +224,7 @@
 entries in the
 .Xr man.conf 5
 file, or with
-.Pa /usr/share/man : Ns Pa /usr/X11R6/man : Ns Pa /usr/local/man
+.Pa /usr/share/man
 by default.
 Within each of these, directories are searched in the order provided.
 Within each directory, the search proceeds according to the following
--- man.conf.5
+++ man.conf.5
@@ -113,8 +113,6 @@
 file at all.
 .Bd -literal -offset indent
 manpath /usr/share/man
-manpath /usr/X11R6/man
-manpath /usr/local/man
 .Ed
 .Sh SEE ALSO
 .Xr apropos 1 ,
--- mandoc.1
+++ mandoc.1
@@ -1016,7 +1016,7 @@
 When running with
 .Fl W Cm base ,
 the search is restricted to the base system, by default to
-.Pa /usr/share/man : Ns Pa /usr/X11R6/man .
+.Pa /usr/share/man .
 This path can be configured at compile time using the
 .Dv MANPATH_BASE
 preprocessor macro.
@@ -1033,7 +1033,7 @@
 environment variable, the
 .Xr man.conf 5
 file and falling back to the default of
-.Pa /usr/share/man : Ns Pa /usr/X11R6/man : Ns Pa /usr/local/man ,
+.Pa /usr/share/man ,
 also configurable at compile time using the
 .Dv MANPATH_DEFAULT
 preprocessor macro.
--- mandoc.db.5
+++ mandoc.db.5
@@ -189,13 +189,6 @@
 .Bl -tag -width /usr/share/man/mandoc.db -compact
 .It Pa /usr/share/man/mandoc.db
 The manual page database for the base system.
-.It Pa /usr/X11R6/man/mandoc.db
-The same for the
-.Xr X 7
-Window System.
-.It Pa /usr/local/man/mandoc.db
-The same for
-.Xr packages 7 .
 .El
 .Pp
 A program to dump
--- NEWS
+++ NEWS
@@ -612,7 +612,7 @@
  * roff(7): Implement \\$* (interpolate all arguments).
  * roff(7): Parse and ignore \, and \/ (italic corrections).
  * When there is no -m, no -M, no MANPATH and no /etc/man.conf,
-   fall back to /usr/share/man:/usr/X11R6/man:/usr/local/man.
+   fall back to /usr/share/man.
  * man(1): Give manuals in purely numerical sections priority over
    manuals of the same name in sections with an alphabetical suffix.
  * man.cgi(8): Support "header.html" and "footer.html".
--- regress/roff/de/Makefile
+++ regress/roff/de/Makefile
@@ -41,10 +41,10 @@
 	rm -f TH.mandoc_ascii_opt Dd.mandoc_ascii_opt
 
 TH.out_ascii_opt: TH.in
-	/usr/local/bin/nroff -c -man -Tascii ${.ALLSRC} > ${.TARGET}
+	/usr/bin/nroff -c -man -Tascii ${.ALLSRC} > ${.TARGET}
 
 Dd.out_ascii_opt: Dd.in
-	/usr/local/bin/nroff -c -mdoc -Tascii ${.ALLSRC} > ${.TARGET}
+	/usr/bin/nroff -c -mdoc -Tascii ${.ALLSRC} > ${.TARGET}
 
 groff-clean-opt:
 	rm -f TH.out_ascii_opt Dd.out_ascii_opt
--- TODO
+++ TODO
@@ -449,7 +449,7 @@
   by syntax tree nodes.
   loc **  exist *  algo **  size *  imp **
 
-- formatting /usr/local/man/man1/latex2man.1 with groff and mandoc
+- formatting /usr/share/man/man1/latex2man.1 with groff and mandoc
   reveals lots of bugs both in groff and mandoc...
   reported by bentley@  Wed, 22 May 2013 23:49:30 -0600
 
